src/hb-ot-cff2-table.hh

```diff
@@ -491,8 +492,8 @@ struct cff2
 	if (unlikely (!priv_interp.interpret (privateDicts[i]))) { fini (); return; }
 
 	privateDicts[i].localSubrs = &StructAtOffsetOrNull<CFF2Subrs> (&privDictStr[0], privateDicts[i].subrsOffset);
-	if (privateDicts[i].localSubrs != &Null(CFF2Subrs) &&
-	  unlikely (!privateDicts[i].localSubrs->sanitize (&sc)))
+	if (privateDicts[i].localSubrs.get () != &Null (CFF2Subrs) &&
```

Comment:
Maybe add is_null() to hb_nonnull_ptr_t?

The way these things were intended to be used, one should never compare to null object...

Reply:
Yep, I don't want put comparison to null pool there so we should compare to nullptr itself and that means removing the above line, which, has unintended effect so lets skip this for now.